examples/pytorch/dtgrnn/README.md

```diff
@@ -0,0 +1,71 @@
+# Discrete Temporal Dynamic Graph with recurrent structure
+## DGL Implementation of DCRNN and GaAN paper.
+
+This DGL example implements the GNN model proposed in the paper [DCRNN](https://arxiv.org/abs/1707.01926) and [GaAN](https://arxiv.org/pdf/1803.07294). 
```

Comment:
I suggest using full name of the paper title. For example Diffusion Convolutional Recurrent Neural Network: Data-Driven Traffic Forecasting (DCRNN)